### DISCUSSION TITLE
Bed Leveler Gauge

### DISCUSSION THREAD

**Discussion by puterboy** [jneilliii/OctoPrint-BedLevelVisualizer#587] [Show and tell]
Here is a gauge that I created that will help you exactly dial-in the angles to adjust your bed.
https://www.printables.com/model/366356-prusa-bed-leveler-gauge

![Screenshot 2023-01-11 144542](https://user-images.githubusercontent.com/1993858/211910651-070e7340-c1d7-441e-9228-5b42b59aeae6.png)

![PXL_20230111_201531818](https://user-images.githubusercontent.com/1993858/211910635-02795045-0b45-4232-bd7e-3d7e1b8f3697.jpg)

**Comment by jneilliii** [comment]
nice

**Comment by puterboy** [comment]
It actually works amazingly well - I can get the numbers all down to single digit degree error (can't do any better as the natural fluctuation between unchanged bed measurements is in that range).

It's a lot easier than trying to "guess" what is 30deg vs. 20 deg. etc.

**Comment by Curlynerd** [comment]
Hi, it looks like the link is not working. Did you move this somewhere?

**Comment by puterboy** [comment]
Fixed - I had setup the url wrong - sorry.

**Comment by triton-nereide** [comment]
Wow, cool one!
